dspace/modules/api/src/main/java/org/datadryad/rest/storage/rdbms/OrganizationDatabaseStorageImpl.java

```diff
@@ -193,8 +225,26 @@ protected void deleteObject(StoragePath path) throws StorageException {
     }
 
     @Override
-    protected void updateObject(StoragePath path, Organization organization) throws StorageException {
-        throw new StorageException("can't update an organization");
+    protected void updateObject(StoragePath path, DryadJournalConcept journalConcept) throws StorageException {
+        // we need to compare the new journal concept that was created to any matching existing concepts.
+        // then we need to update the original concept with the new one
+        // then we delete the new one.
+        Context context = null;
+        try {
+            context = getContext();
+            context.turnOffAuthorisationSystem();
+            String name = journalConcept.getFullName();
+            String status = journalConcept.getStatus();
+            context.commit();
+            DryadJournalConcept conceptToUpdate = JournalUtils.getJournalConceptByJournalName(name);
```

Comment:
This `JournalUtils.getJournalConceptByJournalName` usage is guarded against null pointer exception, but there is one in the file PaymentSystemImpl.java in the method  updateJournal() that is not (sorry -- i can't line comment that file in this diff).